replication/client/conn/gomock_reflect_099241593/prog.go

```diff
@@ -0,0 +1,66 @@
+
+package main
```

Comment:
What's the for?

Reply:
This was auto-generated, I assumed it was needed for a particular step, but based on there wasn't any modification to the client it may not be. I think this confirmed it. Let me remove and look into adding a `.gitignore` entry.